test/common/grpc/grpc_client_integration.h

```diff
@@ -122,7 +122,7 @@ class DeltaSotwIntegrationParamTest
 #define DELTA_SOTW_GRPC_CLIENT_INTEGRATION_PARAMS                                                  \
   testing::Combine(testing::ValuesIn(TestEnvironment::getIpVersionsForTest()),                     \
                    testing::Values(Grpc::ClientType::EnvoyGrpc, Grpc::ClientType::GoogleGrpc),     \
-                   testing::Values(Grpc::SotwOrDelta::Sotw, Grpc::SotwOrDelta::Delta))
+                   testing::Values(Grpc::SotwOrDelta::Sotw))
```

Comment:
This would be a test regression?

Reply:
:+1: 

I forgot why I changed this... Reverted